build-source-tarball.sh

```diff
@@ -85,6 +110,10 @@ while :; do
     shift
 done
 
+if [[ -d "$CUSTOM_REF_PACKAGES_DIR" && -d "$CUSTOM_PREVIOUSLY_BUILT_PACKAGES_DIR" ]]; then
+    MAIN_BUILD_ARGS+=( "/p:SkipDownloadingPreviouslyBuiltArtifacts=true" )
```

Comment:
It seems like this assumes that these two options will always be specified together.  Is that your intention?

Reply:
Hmm, yeah, I think I should split this out into two properties.  I started off doing this differently but never fixed that up, thanks.